tests/testsuite/features2.rs

```diff
@@ -1033,12 +1180,40 @@ fn decouple_proc_macro() {
 
     p.cargo("run")
         .env("TEST_EXPECTS_ENABLED", "1")
-        .with_stdout("it is true")
+        .with_stdout_data(str![[r#"
+it is true
+
+"#]])
         .run();
     // Make sure the test is fallible.
     p.cargo("test --doc")
         .with_status(101)
-        .with_stdout_contains("[..]common is wrong[..]")
+        .with_stdout_data(str![[r#"
+
+running 1 test
+test src/lib.rs - (line 4) ... FAILED
+
+failures:
+
+---- src/lib.rs - (line 4) stdout ----
+Test executable failed ([EXIT_STATUS]: 101).
+
+stderr:
+thread 'main' panicked at src/lib.rs:7:3:
+assertion `left == right` failed: common is wrong
+  left: false
+ right: true
+[NOTE] run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Comment:
There's a backtrace in CI, but I don't think including it here is necessary. However, I'm unsure how to make it work without the backtrace.

Any ideas, @weihanglo?

Reply:
Ah, this shouldn't be an issue anymore. I just realized I can simply use `...` instead.